### PR TITLE
make profile page usable on mobile

### LINKED ISSUE #29340: Profile page is not usable on mobile
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. Open the profile with a mobile phone

### Expected behaviour
Looks at least okay

### Actual behaviour
Bildschirmfoto am 2021-10-20 um 16 42 11

### PR DESCRIPTION
**Pull request by szaimen**
Fix https://github.com/nextcloud/server/issues/29340

| Before | After |
|---|---|
| 192 168 146 130_8443_index php_u_admin(Moto G4) (1) | 192 168 146 130_8443_index php_u_admin(Moto G4) |

Signed-off-by: szaimen <szaimen@e.mail.de>

<details>
<summary>For my own testing</summary>

```
docker run -it \
-e SERVER_BRANCH=fix/29340/fix-mobile-profilepage \
-p 8443:443 \
-e TRUSTED_DOMAIN=192.168.146.130 \
--name nextcloud-easy-test \
ghcr.io/szaimen/nextcloud-easy-test:latest
```

</details>

### CODE REVIEW

**Review by artonge — APPROVED** [review]

File: core/src/views/Profile.vue [hunk 1]
```diff
@@ -506,6 +506,47 @@ $content-max-width: 640px;
 	}
 }
 
+@media only screen and (max-width: 1000px) {
```
Comment (juliushaertl) [inline core/src/views/Profile.vue:509]: Rather use 1024 as in https://github.com/nextcloud/server/blob/3be9d3ca8fca4fb743a4d2f2ffe44a45fa9ffa6e/core/css/variables.scss#L119

Unfortunately using the scss variable won't work here unless we import the variables directly.

Comment (szaimen) [reply core/src/views/Profile.vue]: done

**Review by Pytal — APPROVED** [review]
👌👌👌

**Comment by szaimen** [comment]
I've just rebased the PR

**Pull request merged**